frontend/src/components/layout/Layout.module.scss

```diff
@@ -276,49 +276,58 @@ $toastify-toast-close-button: var(--toast-close-button);
 .footer {
   background-color: black;
   color: $color-white;
-  font-weight: 700;
-  display: flex;
-  align-items: flex-start;
-  flex-direction: column;
 
-  @media screen and #{$mq-md} {
-    flex-direction: row;
-    align-items: center;
-  }
-
-  .mozilla-logo {
-    padding: $spacing-xl;
-  }
-
-  .meta {
+  .links {
+    font-weight: 700;
     display: flex;
+    align-items: flex-start;
     flex-direction: column;
-    padding: $spacing-lg;
-    list-style-type: none;
-    gap: $spacing-md;
 
     @media screen and #{$mq-md} {
       flex-direction: row;
       align-items: center;
-      margin-left: auto;
     }
 
-    a {
-      padding: $spacing-md;
+    .mozilla-logo {
+      padding: $spacing-xl;
+    }
 
-      img {
-        display: inline-block;
+    .meta {
+      display: flex;
+      flex-direction: column;
+      padding: $spacing-lg;
+      list-style-type: none;
+      gap: $spacing-md;
+
+      @media screen and #{$mq-md} {
+        flex-direction: row;
+        align-items: center;
+        margin-left: auto;
       }
 
-      &:hover {
-        color: $color-light-gray-40;
+      a {
+        padding: $spacing-md;
 
         img {
-          opacity: 0.8;
+          display: inline-block;
+        }
+
+        &:hover {
+          color: $color-light-gray-40;
+
+          img {
+            opacity: 0.8;
+          }
         }
       }
     }
   }
+
+  .stock-photo-disclaimer {
+    display: block;
+    padding: $spacing-md;
+    text-align: center;
+  }
```

Comment:
The styling above wasn't changed, just mostly moved from `.footer` to `.links`. The `.stock-photo-disclaimer` styles are actually new.